src/preload-hover.js

```diff
@@ -32,34 +32,34 @@ export default class PreloadHover {
       let timer;
       let uniqueLinks = uniqBy(links, 'href');
```

Comment:
`uniqueLinks` is no longer used and can be removed. We can also remove the `lodash.uniqby` package.